webapp/src/services/api.ts

```diff
@@ -18,8 +18,8 @@ const responseToData =
     try {
       const response = await responsePromise(arg);
       return { data: await response.json() };
-    } catch {
-      return { error: { message } };
+    } catch (error) {
+      return { error: { message: `${message}\n${error}` } };
```

Comment:
Oups! There was an issue with my suggestion. It should have been tested before committing it. The implicit conversion from error to string adds an `Error: ` in front of the error message. You were right to use the `.message` attribute!
```suggestion
      return { error: { message: `${message}\n${(error as Error).message}` } };
```